packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -207,8 +207,8 @@ export const render = async (req, res) => {
 const renderApp = (args) => {
     const {req, res, location, routes, appState, error, App} = args
 
-    const ssrOnly = 'mobify_server_only' in req.query
-    const prettyPrint = 'mobify_pretty' in req.query
+    const ssrOnly = 'mobify_server_only' in req.query || 'server_only' in req.query
+    const prettyPrint = 'mobify_pretty' in req.query || 'pretty_print' in req.query
```

Comment:
When you do this you are reserving the use of `server_only` and `pretty_print` `GET` params for the SDK, entirely. We should prefix these with _something_ to indicate that they are for internal use, establish that as a pattern and follow it for _all_ GET params we reserve like this forever. That's why the previous thing was prefixed with `mobify`.

Reply:
I agree. Using the word reserved here is a little misleading as most people would immediately think of reserved variable names in a language like you can't use `class` as a variable name. But in this scenario you can still use these variable names in both your express app endpoints and third party api endpoints, but there will be side-effects. 

My goal is to not have too many words that comprise the flag name (yes that is an arbitrary goal, but it can be annoying if it's too long). 

Do you have any suggestions, maybe prefix with `pk_` for pwa-kit?

Reply:
maybe `ssr_`